db/active_replicator_config.go

```diff
@@ -85,6 +85,10 @@ type ActiveReplicatorConfig struct {
 	// Map corresponding to db.replications.[replicationID] in Sync Gateway's expvars.  Populated with
 	// replication stats in blip_sync_stats.go
 	ReplicationStatsMap *base.DbReplicatorStats
+
+	// This can be used in a test in order to set ignoreNoConflicts to false. Allows for simulating an earlier Sync
+	// Gateway version where allow_conflicts is set to false.
+	DisableIgnoreNoConflicts bool
```

Comment:
In general I don't like test-only changes to production code - I think I'd prefer flagging this as a fix requiring functional testing instead of unit testing, because of the cross-version dependencies.

Reply:
Keeping this in for now so I can easily test changes based on above comment. Will remove when happy with the method used